cmd/keycmd/transfer.go

```diff
@@ -732,6 +763,13 @@ func exportFromP(
 		}
 		return err
 	}
+	pContext := wallet.P().Builder().Context()
+	pFeeCalculator := avagofee.NewDynamicCalculator(pContext.ComplexityWeights, pContext.GasPrice)
+	txFee, err := pFeeCalculator.CalculateFee(unsignedTx)
+	if err != nil {
+		return err
+	}
+	ux.Logger.PrintToUser("P-Chain Paid fee: %.9f", float64(txFee)/float64(units.Avax))
```

Comment:
would recommend to put AVAX at the end

Reply:
its to differentiate when users pay with their blockchain gas token. Maybe AVAX (P-Chain)?

Reply:
We can add that👍